src/plugins/unified_search/public/filter_bar/filter_bar.tsx

```diff
@@ -50,6 +55,7 @@ const FilterBarUI = React.memo(function FilterBarUI(props: Props) {
         indexPatterns={props.indexPatterns!}
         timeRangeForSuggestionsOverride={props.timeRangeForSuggestionsOverride}
         hiddenPanelOptions={props.hiddenPanelOptions}
+        readOnly={props.isDisabled}
```

Comment:
filter items already supported this prop:

https://github.com/elastic/kibana/blob/8866a8fa16db1c0db68790417cee0f4ef87b0a93/src/plugins/unified_search/public/filter_bar/filter_item/filter_items.tsx#L29

I think ideally we should support both readonly and disabled because they can be a bit different visually and functionally. But as a quick improvement for the team needs I called the props everywhere `isDisabled` because underneath we used `disabled` state for all controls, except this place where we already had `readOnly` and quite a bit of custom code inside (more then just passing disabled prop)